packages/interwovenkit-react/src/pages/wallet/tabs/portfolio/PositionSection.module.css

```
@@ -203,6 +203,58 @@
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.perpToken {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  gap: 6px;
+
+  min-width: 0;
```

Stylelint (17.10.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)